Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -1760,6 +1760,45 @@ struct BuildCommandTestCases {
             data.config == .release
         }
     }
+
+    @Test(
+        .requireHostOS(.macOS),
+        .tags(
+            .Feature.CommandLineArguments.BuildSystem,
+            .Feature.CommandLineArguments.Configuration,
+        ),
+        arguments: getBuildData(for: [.swiftbuild]),
+    )
+    func trivialPackageResources(
+        data: BuildData,
+    ) async throws {
+        let buildSystem = data.buildSystem
+        try await fixture(name: "Miscellaneous/TIF") { fixturePath in
+            let result = try await build(
+                [],
+                packagePath: fixturePath,
+                configuration: data.config,
+                cleanAfterward: false,
+                buildSystem: buildSystem,
+            )
+            try #require(result.binContents.contains("TIF_TIF.bundle"))
+            let contentsDir = result.binPath.appending("TIF_TIF.bundle", "Contents")
+            let bundleResourceDir = contentsDir.appending("Resources")
+
+            let bundleResources = try localFileSystem.getDirectoryContents(bundleResourceDir)
+            #expect(bundleResources.contains("some.txt"))
+            #expect(bundleResources.contains("Assets.car"))
+            #expect(bundleResources.contains("SomeAlert.nib"))
+
+            // Check that the Info.plist of the resource bundle looks reasonable.  In particular, it shouldn't have a CFBundleExecutable key, since it's a codeless bundle.
+            let infoPlistPath = contentsDir.appending("Info.plist")
```

Comment:
Suggestion: SwiftPM has no control over the contents of the info.plist file. As a result, changes to the SwiftBuild service can impact this test. Can we move the intent of this test to SwiftBuild?

Reply:
SwiftPM does have control depending on what build settings it passes down to swift-build.

This PR is not test only, it reverts a line of code that changed the structure of the Info.plist in resource bundles.